### PR TITLE
Add a mailmap 🎄🎅

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14725)

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Gregory Terzian <gterzian@users.noreply.github.com>
+Guillaume Gomez <guillaume1.gomez@gmail.com>
+hgentry <harrison@gentryville.net> <harrison.c.gentry@gmail.com>
+Hugo Thiessard <hugo.thiessard@etu.u-bordeaux.fr> <hugo.thiessard@opmbx.org>
+HyunJune Kim <hyunjune.kim@samsung.com> <hykim0777@gmail.com>
+Ilyong Cho <ilyoan@gmail.com>
+Jacob Parker <j3parker@csclub.uwaterloo.ca> <j@cob.xxx>
+James Graham <james@hoppipolla.co.uk>
+Jansen Jan <farodin91@sek-server.de> <farodin91@googlemail.com>
+Jason Williams <jase.williams@gmail.com>
```
Comment (emilio) [inline .mailmap]: Shoudl we just remove the single entries?

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Clark Gaebel <cgaebel@mozilla.com> <cg.wowus.cg@gmail.com>
+Connor Brewster <connor.brewster@eagles.oc.edu> <brewsterc@my.caspercollege.edu>
+Daniel Robertson <dan.robertson@anidata.org> <danlrobertson89@gmail.com>
+Daniel Robertson <dan.robertson@anidata.org> <drobertson@tripwire.com>
+David Raifaizen <d-raif@hotmail.com> <whoknows>
+Dirkjan Ochtman <dirkjan@ochtman.nl> <d.ochtman@activevideo.com>
+edunham <edunham@mozilla.com> <github@edunham.net>
+Emilio Cobos Álvarez <me@emiliocobos.me> <ecoal95@gmail.com>
+Emilio Cobos Álvarez <me@emiliocobos.me> <emilio@crisal.io>
+Felipe Lacerda <fegolac@gmail.com>
```
Comment (emilio) [inline .mailmap]: This line is duplicated it seems.

File: .mailmap [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Clark Gaebel <cg.wowus.cg@gmail.com> <cgaebel@uwaterloo.ca>
+Clark Gaebel <cgaebel@mozilla.com> <cg.wowus.cg@gmail.com>
+Connor Brewster <connor.brewster@eagles.oc.edu> <brewsterc@my.caspercollege.edu>
+Daniel Robertson <dan.robertson@anidata.org> <danlrobertson89@gmail.com>
+Daniel Robertson <dan.robertson@anidata.org> <drobertson@tripwire.com>
+David Raifaizen <d-raif@hotmail.com> <whoknows>
+Dirkjan Ochtman <dirkjan@ochtman.nl> <d.ochtman@activevideo.com>
+edunham <edunham@mozilla.com> <github@edunham.net>
+Emilio Cobos Álvarez <me@emiliocobos.me> <ecoal95@gmail.com>
+Emilio Cobos Álvarez <me@emiliocobos.me> <emilio@crisal.io>
```
Comment (emilio) [inline .mailmap]: Can you put emilio@crisal.io as primary email? I don't use the other that much :)

File: .mailmap [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+Per Lundberg <perlun@gmail.com> <per.lundberg@ecraft.com>
+Pierre Louis Aublin <pierre-louis.aublin@inria.fr> <pierrelouis.aublin@gmail.com>
+Prabhjyot Singh Sodhi <prabhjyotsingh95@gmail.com> <prabhyotsingh95@gmail.com>
+Ravi Shankar <wafflespeanut@gmail.com>
+Rohan Prinja <rohan.prinja@gmail.com> <rohan.prinja@samsung.com>
+Rohan Prinja <rohan.prinja@gmail.com> <rohan@cs.unc.edu>
+Rohit Zambre <rzambre@uci.edu> <RZ13@dhcp-v007-158.mobile.uci.edu>
+Rohit Zambre <rzambre@uci.edu> <RZ13@dhcp-v058-081.mobile.uci.edu>
+Roman Klauke <romaaan.git@gmail.com> <romankl@users.noreply.github.com>
+rwakulszowa <rwakulszowa1@gmail.com> <rwa@localhost.localdomain>
```
Comment (emilio) [inline .mailmap]: Should we keep `localhost` entries?

File: .mailmap [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+bors-servo <release+servo@mozilla.com> <lbergstrom+bors@mozilla.com>
+bors-servo <release+servo@mozilla.com> <metajack+bors@gmail.com>
+Brian Anderson <banderson@mozilla.com> <andersrb@gmail.com>
+Brian J. Burg <burg@cs.washington.edu>
+Clark Gaebel <cg.wowus.cg@gmail.com> <cgaebel@uwaterloo.ca>
+Clark Gaebel <cgaebel@mozilla.com> <cg.wowus.cg@gmail.com>
+Connor Brewster <connor.brewster@eagles.oc.edu> <brewsterc@my.caspercollege.edu>
+Daniel Robertson <dan.robertson@anidata.org> <danlrobertson89@gmail.com>
+Daniel Robertson <dan.robertson@anidata.org> <drobertson@tripwire.com>
+David Raifaizen <d-raif@hotmail.com> <whoknows>
```
Comment (emilio) [inline .mailmap]: `whoknows`? Should we keep the full name for dubious emails?

```mailmap
David Raifaizen <d-raif@hotmail.com> David Raifaizen <whoknows>
```

**Review by emilio — COMMENTED** [review]
r=me with or without those addressed. Looks fine to me.

File: .mailmap (hunk 5 above)
Comment (nox) [inline .mailmap]: Good idea.

File: .mailmap (hunk 3 above)
Comment (nox) [inline .mailmap]: Ok.

File: .mailmap (hunk 2 above)
Comment (nox) [inline .mailmap]: Right.

File: .mailmap (hunk 1 above)
Comment (nox) [inline .mailmap]: No, that fixes the name.

File: .mailmap (hunk 4 above)
Comment (nox) [inline .mailmap]: Doesn't hurt IMO.

**Pull request merged**